lib/quby/questionnaires/entities/flag.rb

```diff
@@ -18,6 +18,16 @@ def initialize(key:,
         def if_triggered_by(answer_flags)
           yield if answer_flags[key] == trigger_on
         end
+
+        def to_codebook(_options = {})
+          output = []
+          output << "#{key} flag"
+          output << " 'true' - #{description_true}"
+          output << " 'false' - #{description_false}"
+          output << " '' (leeg) - Vlag niet ingesteld, informatie onbekend"
```

Comment:
Ik probeer dat niet te zeggen, je stelling is waar.  Het is een momentopname. Stel dat je een flag maakt 'client rookt', trigger on true shows hidden by default rookvragen, maar de lijst verandert op een gegeven moment naar trigger on false, hides rookvragen (de correcte implementatie omdat je de rookvragen ook in het nil geval wil laten zien). Dan verandert dat 'default' gegeven in de codebook. Verder suggereert default dat het kiezen om het leeg te maken een bewuste keuze is, in plaats van dat dat eigenlijk de default is. Het trigger_on gedrag is een iets te implementatiespecifiek gegeven, ik laat het liever aan de codeboek/export gebruiker over om uit de invullingen zelf icm de vlaginhoud af te leiden of alle verwachte vragen zijn beantwoord.